pydy/codegen/ode_function_generators.py

```diff
@@ -885,11 +898,11 @@ def __init__(self, *args, **kwargs):
 
     def define_inputs(self):
 
-        if self.specifieds is None:
-            self.inputs = self.coordinates + self.speeds + self.constants
-        else:
-            self.inputs = (self.coordinates + self.speeds + self.specifieds
-                           + self.constants)
+        specifieds = []
+        if self.specifieds is not None:
+            specifieds = self.specifieds
+        self.inputs = chain(self.coordinates, self.speeds, specifieds,
```

Comment:
Oh yeah, but this code has never been in an official release. So we don't need deprecation. So what is the reason for changing the type of this attribute?